### PR TITLE
jlh bug fixes

### PR DESCRIPTION
**Pull request by jeremyhermann**

### CODE REVIEW

File: app/views/invitations/show.html.erb [hunk 1]
```diff
@@ -33,7 +33,7 @@
   <div class='centered'>
     <a class='logo' href="http://www.zangzing.com"></a>
     <div class='title'>Congratulations!</div>
-    <div class='sub-title'>You’ve been invited to join ZangZing.</div>
+    <div class='sub-title'>You&rsquo;ve been invited to join ZangZing.</div>
```
Comment (alvarezm50) [inline app/views/invitations/show.html.erb:36]: Was this the cause of the unexplained exclamation marks?

**Comment by alvarezm50** [comment]
Looks good to me

File: app/views/invitations/show.html.erb (hunk 1 above)
Comment (jeremyhermann) [inline app/views/invitations/show.html.erb:36]: nope. haven't figured out the ! issue yet

File: app/models/like.rb [hunk 2]
```diff
@@ -46,15 +46,30 @@ def self.add( user_id, subject_id, subject_type )
...
+
+        when ALBUM, 'album' then
+          album = Album.find(subject_id)
+          notify_user_ids = album.viewers(false)
+          notify_user_ids.reject!{ |id| id == user_id } # don't notify the user who does the like
+
+          notify_user_ids.each do |notify_user_id|
+              ZZ::Async::Email.enqueue( :album_liked, user_id, album.id, notify_user_id )
+          end
+
```
Comment (bowenli) [inline app/models/like.rb:60]: Does this address #4311 - Send like email out to the group? Seems like dup of #4557

Comment (jeremyhermann) [inline app/models/like.rb:60]: yes -- good catch. same feature in 2 milestones. i marked the one assigned to you as 'duplicate'

**Pull request merged**